backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/ProcessDownVmCommand.java

```diff
@@ -163,11 +172,34 @@ protected void executeCommand() {
                 vdsManager.unpinVmCpus(getVm().getId());
             }
         }
-
+        removeNumaPinningForDedicated();
```

Comment:
Will be probably needed in more places, the best would be having the below function in common place.